components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1598,4 +1600,13 @@ void RewardsServiceImpl::Log(ledger::LogLevel level, const std::string& text) {
   VLOG(level) << text;
 }
 
+bool RewardsServiceImpl::CheckImported() {
+//  PrefService* prefs = profile_->GetOriginalProfile()->GetPrefs();
+//  const uint64_t pinned_item_count = prefs->GetUint64(kBravePaymentsPinnedItemCount);
+//  prefs->SetUint64(kBravePaymentsPinnedItemCount, 0);
```

Comment:
This is likely commented out because in your branch `kBravePaymentsPinnedItemCount` is not defined, right?

Reply:
correct, will enable when https://github.com/brave/brave-core/pull/736 is merged